content/en/docs/release-oversight/improving-ci-signal.md

```diff
@@ -68,18 +68,24 @@ Our corpus/history (n) will be well over 250, so use that line.  Our sample size
 If you are making a large change, and are concerned about breaking job flavors that are not covered by your repository's usual presubmits, consider running [`/payload ...` testing](/docs/release-oversight/payload-testing) before merging your change.
 
 #### Who is watching for payload regressions and opening reverts?
-For now, the [Technical Release Team](/docs/release-oversight/the-technical-release-team) watches payload results on the current release branch.  They will open the revert PRs and engage the involved teams (any team that contributed a PR to the payload that regressed).
+After your code merges, your work isn't complete until it's part of two accepted payloads. This ensures your code works, and is upgradeable. If a payload fails after your change merges, check if your code is the cause and, if so, revert that change.
+
+The [Technical Release Team](/docs/release-oversight/the-technical-release-team) acts as a safety net, identifying and reverting breaking changes that aren’t already found.
 
 #### How will we be contacted if our PR is part of a regressed payload?
-TRT will ping the relevant team slack aliases in #forum-release-oversight.  
+TRT will ping the relevant team slack aliases in #forum-release-oversight.
 
 #### Who approves the revert for merge?
-In order to ensure they are included in the conversation/resolution, we would like the team that delivered the original PR to /approve the reversion.  However, if they are unavailable or unresponsive this may be escalated to the staff-engineering team to ensure we do not remain in a regressed state longer than necessary.
+TRT has authority to apply the approved label to any revert in the OpenShift org, even for repositories they are not in OWNERS.  They will notify the original author on Slack and in the revert PR, but to avoid staying regressed any longer than necessary, will not wait for acknowledgement. This also means if you're reverted, you don't need to respond until your normal working hours.
```

Comment:
Maybe soften it a little by changing the order a bit:

`
TRT will attempt to notify the original author on Slack and in the revert PR.  But to avoid staying regressed any longer than necessary, they have authority to apply the approved label to any revert in the OpenShift org, even for repositories they are not in OWNERS.  When the signal is clear regarding the source of a regression TRT has a mandate to restore payloads to healthy state as quickly as possible.  This also means if you're reverted, you don't need to respond until your normal working hours.
`